src/inausoft.netCLI.Tests/Mocks/Command1.cs

```diff
@@ -9,9 +9,6 @@ class Command1
         [OptionAttribute("stringOption")]
         public string StringOption { get; set; }
 
-        [OptionAttribute("pathOption")]
-        public string PathOption { get; set; }
-
         [OptionAttribute("intOption")]
```

Comment:
I would use `Option` instead of `OptionAttribute`.

Reply:
Agreed.